### PR TITLE
Use leaflet in maps

### PR DESCRIPTION
**Pull request by xkjyeah**
Help to review:

- beeline/leaflet
- Changes to components under beeline/

There are quite a number of things to clean up and I'll make them in a separate PR

### CODE REVIEW

**Comment by xkjyeah** [comment]
Oops. I think I need to
1. Copy the changes from `www` to `static`
2. Spin off the webpack config changes into a separate PR

File: beeline/leaflet/CdGmap.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
+/**
+  * Quick-and-dirty maps on Cordova implementation. Falls back to Google-maps
```
Comment (LoneRifle) [inline beeline/leaflet/CdGmap.js]: But... we're not using Cordova anymore.

Comment (xkjyeah) [reply beeline/leaflet/CdGmap.js]: cd- is just to choose a different prefix than what's already been taken (ui-)

Comment (LoneRifle) [reply beeline/leaflet/CdGmap.js]: Alright... `le-`?

**Comment by joelchoo** [comment]
Labelled with shelved for future reference so we're closing this issue

**Pull request closed without merging**